substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/jfr/TestThreadParkEvents.java

```diff
@@ -79,6 +80,11 @@ private static void validateEvents(List<RecordedEvent> events) {
         boolean parkUnparkFound = false;
 
         for (RecordedEvent event : events) {
+            List<RecordedFrame> frames = event.getStackTrace().getFrames();
+            assertTrue(frames.size() > 0);
+            System.out.println(frames.getFirst().getMethod().getName());
```

Comment:
Is this on purpose or is it a leftover from debugging?

Reply:
Yes, that should have been replaced with `checkStackTraceTrimming` . Thanks for spotting that.